aiida_optimade/routers/structures.py

```diff
@@ -19,11 +19,12 @@
 from aiida_optimade.mappers import StructureMapper
 from aiida_optimade.models import StructureResource
 from aiida_optimade.routers.utils import get_entries, get_single_entry, close_session
-
+from aiida_optimade.config import CONFIG
```

Comment:
I would move this up to be the first import line from `aiida_optimade` - just to keep the import lines ordered alphabetically.
This is a formatting thing, not a requirement.

Reply:
Thanks! I open https://github.com/aiidateam/aiida-optimade/pull/388 to use reorder importers pre-commit hook.